TramsDataApi/Gateways/AcademyConversionProjectGateway.cs

```diff
@@ -103,6 +104,15 @@ private static IQueryable<AcademyConversionProject> FilterByStatus(IEnumerable<s
 
             return queryable;
         }
+        private static IQueryable<AcademyConversionProject> FilterByRegion(IEnumerable<int?> regions, IQueryable<AcademyConversionProject> queryable)
+        {
+            if (regions != null && regions!.Any())
```

Comment:
This will cause a request to the DB if `queryable` is an EF query - multiple enumeration. Linq will deal with empty sets, so this is not needed.

Reply:
Did the above and have broken a test as a result, am resolving as we speak.

Reply:
Actually I put it back, it's correct.